cms/templates/base.html

```diff
@@ -37,6 +37,7 @@
     %>
 
     <script type="text/javascript" src="${static.url(jsi18n_path)}"></script>
+    <link rel="icon" type="image/x-icon" href="${static.url('images/favicon.ico')}" />
```

Comment:
Should we be using `{% favicon_path %}` since it's already defined [here](https://github.com/mitodl/edx-platform/blob/mitx/ginkgo/lms/envs/common.py#L894)?

Reply:
It is only defined in lms

Reply:
Can we add it?

Reply:
Looks good to me,
But in lms we doing something like `  <link rel="icon" type="image/x-icon" href="${static.url(static.get_value('favicon_path', settings.FAVICON_PATH))}" />
`
i think edx should make this setting shared